### PR TITLE
Fixing "ASP.NET Core on Nano Server" tutorial

### PR DESCRIPTION
**Pull request by kichalla**
@shirhatti @Rick-Anderson

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Wow, this looks great. @shirhatti  do you want to review before I merge this?

**Comment by shirhatti** [comment]
@kichalla Can you update the applicationHost.config that accompanies this article https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/nano-server/applicationHost.config

File: aspnet/tutorials/nano-server.rst [hunk 1]
```diff
@@ -40,55 +40,75 @@ Open an elevated PowerShell window to add your remote Nano Server instance to yo
...
+
+.. code:: ps1
+
+  copy \\<nanoserver-ip-address>\AspNetCoreSampleForNano\aspnetcore_schema.xml C:\windows\system32\inetsrv\config\schema
+  copy \\<nanoserver-ip-address>\AspNetCoreSampleForNano\aspnetcore.dll C:\windows\system32\inetsrv
+
+  # Backup existing applicationHost.config before installing ANCM
   copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_BeforeInstallingANCM.config
 
   Import-Module IISAdministration
```
Comment (shirhatti) [inline aspnet/tutorials/nano-server.rst]: Can you move this inline powershell script back into this file https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/nano-server/enable-ancm.ps1

**Comment by kichalla** [comment]
> @kichalla Can you update the applicationHost.config that accompanies this article https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/nano-server/applicationHost.config

Do we need this anymore anymore? Like we have script which take care of installing ANCM and the New-IISSite commandlet modifes this for us.

**Comment by kichalla** [comment]
:up: :date:

**Comment by shirhatti** [comment]
> Do we need this anymore? Like we have script which take care of installing ANCM and the New-IISSite commandlet modifes this for us.

My only motivation for having it was since the script is not idempotent and you run the risk of borking your IIS install if you re-run it. Especially since if you run the script a second time you will mess up the back-up you've created as well.

I've personally found it useful in the past 😄 but it's your call on whether to update/delete applicationHost.config

**Comment by kichalla** [comment]
I see. The only reason I would hesitate to keep it now is that, when I was working on Nano server I tried to compare the IIS configs of two different installs and noticed that these values were different. So am I not sure if the content of these lines are machine specific.

https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/nano-server/applicationHost.config#L115-L116

**Comment by shirhatti** [comment]
> I see. The only reason I would hesitate to keep it now is that, when I was working on Nano server I tried to compare the IIS configs of two different installs and noticed that these values were different. So am I not sure if the content of these lines are machine specific.

Point taken.
I'll update the PR to delete applicationHost.config before merging in your changes

**Pull request merged**